src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/edgpsi/experimental_detectron_generate_proposals_single_image_kernel_ref.cpp

```diff
@@ -0,0 +1,177 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "experimental_detectron_generate_proposals_single_image_kernel_ref.h"
+#include "kernel_selector_utils.h"
+#include <algorithm>
+#include <string>
+
+namespace kernel_selector {
+
+ParamsKey ExperimentalDetectronGenerateProposalsSingleImageRef::GetSupportedKey() const {
+    ParamsKey k;
+    k.EnableInputDataType(Datatype::F16);
+    k.EnableInputDataType(Datatype::F32);
+    k.EnableOutputDataType(Datatype::F16);
+    k.EnableOutputDataType(Datatype::F32);
+    k.EnableInputLayout(DataLayout::bfyx);
+    k.EnableOutputLayout(DataLayout::bfyx);
```

Comment:
<html>
<body>
<!--StartFragment--><p dir="auto" style="box-sizing: border-box; margin-top: 0px !important; margin-bottom: 16px; color: rgb(201, 209, 217); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(13, 17, 23); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">Could you please add those blocked format for input/output layout? Those are required for dGPU support.</p>

Prec | Layout
-- | --
int8 | fsv16, fsv32, bsv32_fsv32
fp16 | fsv16, bsv32_fsv16
fp32 | fsv16, bsv32_fsv16

<!--EndFragment-->
</body>
</html>

Reply:
we should add it to all the open PRs, right?

Reply:
Yes, please add them for all PR if it doesn't have.

Reply:
done

Reply:
Sorry for confusion. I checked internally and we need full support of  fsv16, fsv32, bsv32_fsv16, bsv32_fsv32, bsv16_fsv16, bsv16_fsv32. I will update comments in other PRs.

Reply:
done

Reply:
If we enabled them, we'll need correspodning unittests. Please add them.

Reply:
as we discussed today - removed this layouts for now